### PR TITLE
Update gnali 1.0 recipe

### PR DESCRIPTION
**Pull request by xialiu71**
Update dependencies for gNALI 1.0

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/gnali/meta.yaml [hunk 1]
```diff
@@ -33,13 +33,19 @@ requirements:
...
+    - libcurl
+    - perl-bio-bigfile
     - pybiomart
     - pandas
     - numpy
     - pysam
     - filelock
     - pyyaml
     - py-bgzip
+    - git >=2
```
Comment (bgruening) [inline recipes/gnali/meta.yaml:48]: Do you really need git as run time dependecy?

Comment (xialiu71) [reply recipes/gnali/meta.yaml:48]: Yes, there's a function that clones a repo (without tagged releases)

**Comment by xialiu71** [comment]
@BiocondaBot please add label

**Comment by xialiu71** [comment]
Sorry, I realized I need to double check something so I'm marking this as a draft

File: recipes/gnali/meta.yaml [hunk 2]
```diff
@@ -33,13 +33,19 @@ requirements:
...
+    - ensembl-vep ==101.*
+    - samtools
+    - tabix
+    - libcurl
+    - perl-bio-bigfile
     - pybiomart
     - pandas
     - numpy
-    - pysam
+    - pysam ==0.15.*
```
Comment (ericenns) [inline recipes/gnali/meta.yaml:44]: I think you can just use `pysam <0.16`

File: recipes/gnali/meta.yaml [hunk 3]
```diff
@@ -33,13 +33,19 @@ requirements:
     - git >=2
   run:
     - python >=3.6
+    - ensembl-vep ==101.*
```
Comment (ericenns) [inline recipes/gnali/meta.yaml:36]: why do you need to specify `101.*`

File: recipes/gnali/meta.yaml (hunk 2 above)
Comment (xialiu71) [reply recipes/gnali/meta.yaml]: Implemented in 53449d9

File: recipes/gnali/meta.yaml (hunk 3 above)
Comment (xialiu71) [reply recipes/gnali/meta.yaml]: Updated as per our discussion in 53449d9

**Comment by xialiu71** [comment]
@BiocondaBot please add label

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**